frontend/projects/ui/src/app/services/form.service.ts

```diff
@@ -89,7 +89,7 @@ export class FormService {
     let group: Record<string, FormGroup | FormArray | FormControl> = {}
     Object.entries(config).map(([key, spec]) => {
       if (spec.type === 'pointer') return
-      group[key] = this.getFormEntry(spec, current ? current[key] : undefined)
+      group[key] = this.getFormEntry(spec, current[key])
```

Comment:
Turns out it was. null is considered a valid value when strict null checking is [enabled](https://stackoverflow.com/questions/57243477/can-an-optional-parameter-be-null-in-typescript), and thus the default of `{}` did not take effect since it was in fact `null`